pkg/ibmvpc/examples/out/lint_out/acl3_overlap_subnets_Lint

```diff
@@ -0,0 +1,22 @@
+In VPC test-vpc1-ky, network ACL acl1-ky rule's indexed 4 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 4, direction: inbound , src: 10.240.20.0/24 , dst: 10.240.10.0/24, conn: all, action: allow
+In VPC test-vpc1-ky, network ACL acl2-ky rule's indexed 0 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 0, direction: outbound , src: 10.240.20.0/24 , dst: 142.0.0.0/8, conn: protocol: icmp, action: allow
+In VPC test-vpc1-ky, network ACL acl2-ky rule's indexed 1 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 1, direction: outbound , src: 10.240.20.0/24 , dst: 10.240.30.0/24, conn: protocol: icmp, action: allow
+In VPC test-vpc1-ky, network ACL acl2-ky rule's indexed 2 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 2, direction: outbound , src: 10.240.20.0/24 , dst: 10.240.10.0/24, conn: all, action: allow
+In VPC test-vpc1-ky, network ACL acl2-ky rule's indexed 5 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 5, direction: inbound , src: 10.240.30.0/24 , dst: 10.240.20.0/24, conn: protocol: tcp, srcPorts: 1-65535, dstPorts: 22-22, action: allow
+In VPC test-vpc1-ky, network ACL acl2-ky rule's indexed 6 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 6, direction: inbound , src: 10.240.10.0/24 , dst: 10.240.20.0/24, conn: all, action: allow
+In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 0 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 0, direction: outbound , src: 0.0.0.0/0 , dst: 10.240.10.0/24, conn: all, action: allow
+In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 1 splits subnets subnet1-ky (10.240.10.0/16), subnet3-ky (10.240.30.0/24). Splitting rule details: index: 1, direction: outbound , src: 10.240.30.0/31 , dst: 10.240.20.0/24, conn: all, action: allow
+In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 2 splits subnet subnet1-ky (10.240.10.0/16). Splitting rule details: index: 2, direction: inbound , src: 10.240.10.0/24 , dst: 0.0.0.0/0, conn: all, action: allow
+In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 3 splits subnets subnet1-ky (10.240.10.0/16), subnet3-ky (10.240.30.0/24). Splitting rule details: index: 3, direction: inbound , src: 10.240.20.0/24 , dst: 10.240.30.0/31, conn: all, action: allow
+________________________________________________________________________________________________________________________________________________________________________________________________________
+
+"Overlapping CIDR ranges between different subnets" issues:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+subnet subnet1-ky of cidr 10.240.10.0/16 and subnet subnet2-ky of cidr 10.240.20.0/24, both from VPC test-vpc1-ky, overlap in 10.240.20.0/24
```

Comment:
isn't the goal to warn about subnets from different VPCs with overlapping ranges? 
I don't think it's possible to configure such subnets in the same VPC.

Reply:
Yes, you are correct. It should not be possible. Redundant code and test removed.

Reply:
please remove also the output file + add another relevant test instead

Reply:
right. output file removed. Added a test in which the overlapping is in part of the range (in which case the output is slightly different)